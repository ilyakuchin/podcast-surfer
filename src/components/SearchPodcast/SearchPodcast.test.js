import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faSearch } from "@fortawesome/free-solid-svg-icons";

configure({ adapter: new Adapter() });

describe("SearchPodcast component test", () => {
  test("should render properly", () => {
    const wrapper = shallow(
      <div>
        <form>
          <input type="text" placeholder="Search.." name="search" />
          <button type="submit">
            <FontAwesomeIcon icon={faSearch} />
          </button>
        </form>
        <ul></ul>
      </div>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
