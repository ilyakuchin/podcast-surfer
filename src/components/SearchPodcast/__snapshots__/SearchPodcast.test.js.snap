// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchPodcast component test should render properly 1`] = `ShallowWrapper {}`;
